### ISSUE TITLE
Tutorial on hosting your own Sour server

### ISSUE THREAD

**Issue by eris-webserv** [cfoust/sour#16]
It would be nice if people knew how to host their own server. Yknow, for 1v1s or testing.

**Comment by cfoust** [comment]
That's a great idea! I will think about this 🤔

**Comment by ProjectRobinson** [comment]
Yes! I keep trying to work on it myself, but am getting stuck. Willing to pay someone to help me learn a little more on deploying this if able. 

Thank you!

**Comment by cfoust** [comment]
What are you getting stuck on? No payment necessary, just let me know and I'll help ya!
